test/parallel_api/ranges/std_ranges_sort.pass.cpp

```diff
@@ -0,0 +1,45 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "std_ranges_test.h"
+
+std::int32_t
+main()
+{
+#if _ENABLE_STD_RANGES_TESTING
+    using namespace test_std_ranges;
+    namespace dpl_ranges = oneapi::dpl::ranges;
+
+    test_range_algo<0>{}(dpl_ranges::sort, std::ranges::sort, std::ranges::less{});
+    test_range_algo<0>{}(dpl_ranges::stable_sort, std::ranges::stable_sort, std::ranges::less{});
+
+    test_range_algo<1>{}(dpl_ranges::sort, std::ranges::sort, std::ranges::less{}, proj);
```

Comment:
The generated data is already sorted. It might be better to adjust it for future sorting somehow.

Reply:
Yes, it is an additional vote for adding a new functional abstraction  - a kind of data generator....

Reply:
Well... I suppose the high/middle layers can do nothing and the test will succeed. This is a peculiar scenario, but I still believe the data for sort should not be already sorted.

Reply:
I've rechecked the test: it sorts the same sequence passing `std::ranges::greater` and `std::ranges::less`, so my comment is not relevant/critical anymore, at least for `std::ranges::greater`.